docs/install.md

```diff
@@ -623,6 +658,115 @@ $ kubectl exec my-csi-app-2 -- mount |grep /data
 /dev/ndbus0region0fsdax/5cc9b19e-551d-11e9-a584-928299ac4b17 on /data type xfs (rw,relatime,attr2,dax,inode64,noquota)
 ```
 
+### Automatic node setup
+
+The expectation is that the scripts which bring up nodes can be
+adapted to prepare the PMEM for usage by PMEM-CSI as explained
+earlier. But this might not always be easy.
+
+For the case of converting an existing "raw" namespace to "fsdax" mode
+there is a possibility to do the conversion through a deployed
+PMEM-CSI driver:
+
+1. Install PMEM-CSI in LVM mode without preparing nodes. At this point
+   only the central controller pod will run.
+1. For each node that has one or more raw namespaces that all need
+   to be converted, set the `<driver name>/convert-raw-namespaces` label (usually
+   `pmem-csi.intel.com/convert-raw-namespaces` to `force`.
+1. This will cause the pods of the `pmem-csi-intel-com-node-setup`
+   DaemonSet to run on those nodes. Those pods then will
+   convert the namespaces, create the LVM volume group,
+   remove the `convert-raw-namespaces` label
+   (i.e. the pods will only run once) and add the normal label
+   that enables the PMEM-CSI node driver pods to run.
+1. The normal node driver pods start up and then are
+   ready to provision volumes.
+
+**WARNING**: the raw namespaces will be converted even when they are
+active. If data was stored on them, it will be lost after the
+conversion.
+
+The output of a successful conversion will look like this:
+```
```

Comment:
The log of a successful conversion is useful when comparing against an unsuccessful one because it shows which messages are normal and which aren't.

Reply:
I reworded the section about how to see these logs.